crates/wasm-smith/src/code_builder.rs

```diff
@@ -3171,3 +3409,528 @@ fn table_index<C: Config>(
         .collect::<Vec<_>>();
     Ok(*u.choose(&tables)?)
 }
+
+fn lane_index(u: &mut Unstructured, number_of_lanes: u8) -> Result<u8> {
+    u.int_in_range(0..=(number_of_lanes - 1))
+}
+
+fn arbitrary_v128(u: &mut Unstructured) -> Result<[u8; 16]> {
+    let mut buffer = [0; 16];
+    u.fill_buffer(&mut buffer)?;
+    Ok(buffer)
+}
```

Comment:
[`[u8; 16]` implements `Arbitrary`](https://docs.rs/arbitrary/1.0.1/arbitrary/trait.Arbitrary.html#impl-Arbitrary%3C%27a%3E-for-%5BT%3B%20N%5D), so calls to this function can be replaced with `u.arbitrary()?`

Reply:
Yeah, that's better... removed.